<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>APT Analyzer</class>
 <widget class="QMainWindow" name="APT Analyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1775</width>
    <height>703</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="AddTechnique">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Technique</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Tactics">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>119</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="Techniques">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>119</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TacticsBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="plainText">
     <string>Tactics</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="TechniquesBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Techniques</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="PotentialMatches">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>540</y>
      <width>521</width>
      <height>71</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Potential Matches above 50%: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Evaluate">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>560</y>
      <width>141</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Reports</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteTechnique">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>540</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Technique</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="SoftwareBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>360</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Software</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Software">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>410</y>
      <width>351</width>
      <height>41</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>46</red>
          <green>194</green>
          <blue>126</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>119</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="insertPolicy">
     <enum>QComboBox::InsertAlphabetically</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSoftware">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>570</y>
      <width>351</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedTechniquesText">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>0</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Techniques</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSoftware">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>540</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete Software</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="AddedSoftwareText">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>0</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <property name="plainText">
     <string>Added Software</string>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedTechniquesBox">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>60</y>
      <width>321</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="AddedSoftwareBox">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>60</y>
      <width>301</width>
      <height>471</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>35</red>
          <green>39</green>
          <blue>41</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="WindowText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>144</red>
          <green>147</green>
          <blue>147</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearAll">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>580</y>
      <width>631</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear All</string>
    </property>
   </widget>
   <widget class="QLabel" name="progress">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>130</y>
      <width>341</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="updating">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>441</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ResetButton">
    <property name="geometry">
     <rect>
      <x>751</x>
      <y>490</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider_bar">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>575</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>25</red>
          <green>229</green>
          <blue>26</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>38</red>
          <green>162</green>
          <blue>105</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>39</red>
          <green>119</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="minimum">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="slider_text">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>569</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="DescriptionBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="updating_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="Groups_List">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>521</width>
      <height>531</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="autoExpandDelay">
     <number>-1</number>
    </property>
    <property name="indentation">
     <number>34</number>
    </property>
    <property name="uniformRowHeights">
     <bool>false</bool>
    </property>
    <column>
     <property name="text">
      <string>GROUPS</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="HTML">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>560</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open HTML Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="GroupPDF">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>560</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open Group Report</string>
    </property>
   </widget>
   <widget class="QPushButton" name="PDF">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>560</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Open PDF Report</string>
    </property>
   </widget>
   <zorder>DescriptionBox</zorder>
   <zorder>PotentialMatches</zorder>
   <zorder>DeleteTechnique</zorder>
   <zorder>DeleteSoftware</zorder>
   <zorder>AddedSoftwareText</zorder>
   <zorder>AddedSoftwareBox</zorder>
   <zorder>ClearAll</zorder>
   <zorder>TacticsBox</zorder>
   <zorder>TechniquesBox</zorder>
   <zorder>SoftwareBox</zorder>
   <zorder>Tactics</zorder>
   <zorder>Techniques</zorder>
   <zorder>Software</zorder>
   <zorder>AddTechnique</zorder>
   <zorder>AddSoftware</zorder>
   <zorder>Evaluate</zorder>
   <zorder>AddedTechniquesText</zorder>
   <zorder>AddedTechniquesBox</zorder>
   <zorder>slider_bar</zorder>
   <zorder>slider_text</zorder>
   <zorder>progress</zorder>
   <zorder>updating</zorder>
   <zorder>updating_2</zorder>
   <zorder>HTML</zorder>
   <zorder>GroupPDF</zorder>
   <zorder>PDF</zorder>
   <zorder>Groups_List</zorder>
   <zorder>ResetButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1775</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpdate"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionUpdate">
   <property name="text">
    <string>Update Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
